docs/HOWTO_USE_POSTGRESQL.md

```diff
@@ -3,9 +3,19 @@
 To use Postgresql, follow these steps:
 
 1. Either install docker (recommended) or postgresql on your machine:
-   * If you installed docker run the database using: `docker compose up`
-   * If you installed postgres locally, create the spoke dev database: `psql -c "create database spokedev;"`
-     * Then create a spoke user to connect to the database with `createuser -P spoke` with password "spoke" (to match the credentials in the .env.example file)
+   * If you installed docker run the database using: `docker-compose up`
+   * If you installed postgres locally (or if you already have a local installation of postgres), create the spoke dev database: `psql -c "create database spokedev;"`
+     * Then create a spoke user to connect to the database with `psql -d spokedev -c "create user spoke with password 'spoke';"` (to match the credentials in the .env.example file)
+     * Grant permissions to the new Spoke user:
+       * `psql -d spokedev -c "GRANT ALL PRIVILEGES ON DATABASE spokedev TO spoke;"`
+       * `psql -d spokedev -c "GRANT ALL PRIVILEGES ON schema public TO spoke;"`
+     * Also run the commands from `./dev-tools/create-test-database`:
```

Comment:
This tool won't work for local pg because its set up for docker.

Reply:
Right, I'm not saying to run the script directly but to manually run the commands to create a test database instead.

Reply:
Ahh I see. I would instead have you place those instructions directly in the document (minus the docker stuff). I can see new devs getting confused like I did.

Reply:
I updated the explanation, hopefully this is more clear!